flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/calls/ScalarOperators.scala

```diff
@@ -702,12 +703,15 @@ object ScalarOperators {
   def generateTemporalPlusMinus(
       plus: Boolean,
       nullCheck: Boolean,
+      typeName: SqlTypeName,
       left: GeneratedExpression,
       right: GeneratedExpression)
     : GeneratedExpression = {
 
     val op = if (plus) "+" else "-"
 
+    val AVGDAYS_PRE_MONTH = 30.5
```

Comment:
Is this official SQL behaviour?

Reply:
Is not a standard, I did not find the relevant norms, do you have any suggestions?